config/locales/en.yml

```diff
@@ -195,6 +195,7 @@ en:
         select_distance_unit: "Show distance in %{distance_units_selector}"
         km: km
         miles: miles
+        show_only_nearby: "Location search shows only nearby listings in the list and grid view"
```

Comment:
I'm thinking how clear the behavior will be for admins from this description. I guess we can explain this further in knowledgebase article (have you coordinated an update to that?) if we link to one somewhere here?